examples/kubernetes-pod-exec-time/README.md

```diff
@@ -38,6 +39,17 @@ Deploy the test Pods to the cluster:
 kubectl apply -f seed/pod.yaml
 ```
 
+Create a meter in OpenMeter with the following details:
```

Comment:
We should probably provide YAML meter configs for the open-source OpenMeter.

Reply:
There is an example in the general examples readme.